### ISSUE TITLE
Neon light don't saving

### ISSUE THREAD

**Issue by martinpino92** [TanguyOrtegat/esx_jb_eden_garage2#95]
The neon lights that one installs in lscustom, are not saved when I deposit my vehicle in the garage, please if you can add this to the code and the script would be perfect, thank you very much for the work!

**Comment by TanguyOrtegat** [comment]
you have the native for neons?

**Comment by iTzCrutchie** [comment]
https://runtime.fivem.net/doc/natives/?_0x7619EEE8C886757F

https://runtime.fivem.net/doc/natives/?_0x8C4B92553E4766A5

https://runtime.fivem.net/doc/natives/?_0x2AA720E4287BF269

https://runtime.fivem.net/doc/natives/?_0x8E0A582209A62695

If I get time I can most likely get this added with a PR. Currently at work but have this implemented in my own private garage script not to complicated to put in.

**Comment by martinpino92** [comment]
I really like your garage script, and I would like to continue using it, so if you have a little time do it please, my respect you already have it, and thanks for listening! greetings from Argentina!

// 0x7619EEE8C886757F 
// GetVehicleNeonLightsColour
void _GET_VEHICLE_NEON_LIGHTS_COLOUR(Vehicle vehicle, int* r, int* g, int* b);
Gets the color of the neon lights of the specified vehicle.  
See _SET_VEHICLE_NEON_LIGHTS_COLOUR (0x8E0A582209A62695) for more information 


// 0x8C4B92553E4766A5 
// IsVehicleNeonLightEnabled
BOOL _IS_VEHICLE_NEON_LIGHT_ENABLED(Vehicle vehicle, int index);
indices:  
0 = Left  
1 = Right  
2 = Front  
3 = Back  

_SET_VEHICLE_NEON_LIGHT_ENABLED
// 0x2AA720E4287BF269 
// SetVehicleNeonLightEnabled
void _SET_VEHICLE_NEON_LIGHT_ENABLED(Vehicle vehicle, int index, BOOL toggle);
Sets the neon lights of the specified vehicle on/off.  
Indices:  
0 = Left  
1 = Right  
2 = Front  
3 = Back  


Colour | R | G | B
-- | -- | -- | --
White | 222 | 222 | 255
Blue | 2 | 21 | 255
Electric Blue | 3 | 83 | 255
Mint Green | 0 | 255 | 140
Lime Green | 94 | 255 | 1
Yellow | 255 | 255 | 0
Golden Shower | 255 | 150 | 0
Orange | 255 | 62 | 0
Red | 255 | 1 | 1
Pony Pink | 255 | 50 | 100
Hot Pink | 255 | 5 | 190
Purple | 35 | 1 | 255
Blacklight | 15 | 3 | 255


_GET_VEHICLE_NEON_LIGHTS_COLOUR
// 0x7619EEE8C886757F 
// GetVehicleNeonLightsColour
void _GET_VEHICLE_NEON_LIGHTS_COLOUR(Vehicle vehicle, int* r, int* g, int* b);
Gets the color of the neon lights of the specified vehicle.  
See _SET_VEHICLE_NEON_LIGHTS_COLOUR (0x8E0A582209A62695) for more information

**Comment by TanguyOrtegat** [comment]
will check this when i got time because have less time now :p

**Comment by Samnick86** [comment]
Yes, neons are not saved :-( Please, fix it. Your garage script is excelent!
